src/components/Description/description.jsx

```diff
@@ -0,0 +1,27 @@
+
+export default function Description(){
+    return (
+    <div class='Description'>
+        <h1 class='header'>
+            style={{
+                color: "#000000",
+                fontWeight: 'bold',
+                fontSize: 38
+            }}
+            Description
+            {"\n"}
+        </h1>
+        <Text numberOfLines={3}>
```

Comment:
This should be using the standard p with a max width instead of number of lines

Reply:
What do you mean by standard p? Figma tells that the width is 602px and the height is 17.77px so should I just write width:602 and height:17.77?

Reply:
No, my confusion is around the <Text> tag, I have never seen that used anywhere, so I am confused as to where it is coming from. The width should be reasonable for the text box

Reply:
Maybe 40-50%